config/config.exs

```diff
@@ -28,3 +28,4 @@ import Config
 # here (which is why it is important to import them last).
 #
 #     import_config "#{Mix.env}.exs"
+import_config "#{config_env()}.exs"
```

Comment:
What happens if the env is `:prod` ?

Reply:
Rather than removing this import statement, I added a `config/prod.exs` file.

Considering there now is a specific setup for the `MIX_ENV=test`, I figured it was the simpler solution.

Works out for you ?

Reply:
My bad, libs config files get ignored when used as dependencies, so it's unnecessary.